Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -192,9 +192,9 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate, UITextFieldDele
                 make.leading.equalTo(self.lockImageView.snp_trailing).offset(locationContentInset)
             }
             if readerModeButton.hidden {
-                make.trailing.equalTo(container.snp_trailing)
+                make.trailing.equalTo(container.snp_trailing).offset(-locationContentInset)
             } else {
-                make.trailing.equalTo(self.readerModeButton.snp_leading)
+                make.trailing.equalTo(self.readerModeButton.snp_leading).offset(-locationContentInset)
```

Comment:
we don't really need the offset here as butting up against the readerMode button is OK and won't affect border rendering.